### PR TITLE
Fix for new notifications "content" type.

### PR DESCRIPTION
**Pull request by peteclark-ft**
The notifications pull url will now include the configured query params if there are any.

This release will also involve a service file change adding `?type=all` to the notifications pull url.

### CODE REVIEW

File: feeds/notificationsPullFeed_test.go [hunk 1]
```diff
@@ -106,9 +106,13 @@ func TestNotificationsArePolled(t *testing.T) {
 
 	httpCaller := mockHTTPCaller(t, "tid_pam_notifications_pull_", buildResponse(200, notifications))
 
-	baseUrl, _ := url.Parse("http://www.example.org")
+	baseUrl, _ := url.Parse("http://www.example.org?example=false")
 	sinceDate := "2016-10-28T15:00:00.000Z"
 	f := NewNotificationsFeed("notifications", baseUrl, sinceDate, 10, 1, "", "")
+
+	uri := f.(*NotificationsPullFeed).buildNotificationsURL()
+	assert.Equal(t, "http://www.example.org?example=false&since=2016-10-28T15%3A00%3A00.000Z", uri)
```
Comment (khatton-ft) [inline feeds/notificationsPullFeed_test.go]: This isn't very robust - the implementation might swap since and example around, and the test should still pass. I'd recommend the test case should parse uri and check the two keys have the expected values. (Also, "false" might be a bit iffy as a test value because if anything converted it into a boolean and back, then the zero value would be false anyway).

Comment (peteclark-ft) [inline feeds/notificationsPullFeed_test.go]: Yeah this is reasonable (a bit rushed).
